docker/nginx-seed.conf

```diff
@@ -173,16 +173,18 @@ http {
         set $STYLE "${STYLE} https://cdnjs.cloudflare.com";
         set $STYLE "${STYLE} https://maxcdn.bootstrapcdn.com";
         set $STYLE "${STYLE} https://stackpath.bootstrapcdn.com";
+        set $STYLE "${STYLE} https://fonts.googleapis.com";
 
         set $FONT "font-src 'self' 'unsafe-inline'";
         set $FONT "${FONT} https://cdnjs.cloudflare.com";
         set $FONT "${FONT} https://maxcdn.bootstrapcdn.com";
+        set $FONT "${FONT} https://fonts.gstatic.com/";
 
         set $FRAME "frame-src 'self'";
         set $FRAME "${FRAME} https://recaptcha.google.com/recaptcha/";
         set $FRAME "${FRAME} https://www.google.com/recaptcha/";
 
-        set $IMG "img-src 'self' data:";
+        set $IMG "img-src https: data:";
```

Comment:
What does changing this line from `'self'` to `https:` do? Does it allows an img-src to load from any https link?

Reply:
It allows only images using https since we are using the pre-signed url and it uses https. I can add the AWS_CUSTOM_DOMAIN to the list of allowed https urls and then it will only allow the s3 related ones. Thanks.